.github/workflows/publish-chart.yaml

```diff
@@ -48,8 +54,29 @@ jobs:
           sed -i -e '136s,false,'true',g' ./helm/trivy/values.yaml
           ct lint-and-install --validate-maintainers=false --charts helm/trivy
 
+  update-chart-version:
+    if: github.event_name == 'push'
```

Comment:
Can we add a comment about this case?
IIUC this work will be started after pushing a new tag, right?

Reply:
Will be great to add comments for other jobs.
this will make it easier to understand this workflow

Reply:
done [09b27ec](https://github.com/aquasecurity/trivy/pull/7526/commits/09b27ec355bc2a78775216ff4c7069efe13a6702)